[2023-10-01 09:51:49,602] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: postgres_v1.create_postgres_connection 2021-08-17T00:00:00+00:00 [queued]>
[2023-10-01 09:51:49,981] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: postgres_v1.create_postgres_connection 2021-08-17T00:00:00+00:00 [queued]>
[2023-10-01 09:51:49,990] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-10-01 09:51:49,997] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2023-10-01 09:51:50,002] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-10-01 09:51:50,599] {taskinstance.py:1063} INFO - Executing <Task(PostgresOperator): create_postgres_connection> on 2021-08-17T00:00:00+00:00
[2023-10-01 09:51:50,671] {standard_task_runner.py:52} INFO - Started process 42626 to run task
[2023-10-01 09:51:50,753] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'postgres_v1', 'create_postgres_connection', '2021-08-17T00:00:00+00:00', '--job-id', '2596', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/1_postgresql.py', '--cfg-path', '/tmp/tmp5s3d5jsy', '--error-file', '/tmp/tmpcwm5jofq']
[2023-10-01 09:51:50,771] {standard_task_runner.py:77} INFO - Job 2596: Subtask create_postgres_connection
[2023-10-01 09:51:51,460] {logging_mixin.py:104} INFO - Running <TaskInstance: postgres_v1.create_postgres_connection 2021-08-17T00:00:00+00:00 [running]> on host d4ed655215d1
[2023-10-01 09:51:52,034] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=svastikkka
AIRFLOW_CTX_DAG_ID=postgres_v1
AIRFLOW_CTX_TASK_ID=create_postgres_connection
AIRFLOW_CTX_EXECUTION_DATE=2021-08-17T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-17T00:00:00+00:00
[2023-10-01 09:51:52,044] {postgres.py:69} INFO - Executing: create table if not exists dags_run(
        dt  date,
        dag_id character varying,
        primary key (dt,dag_id)
        )
[2023-10-01 09:51:52,348] {base.py:74} INFO - Using connection to: id: posgres_testing. Host: host.docker.internal, Port: 5432, Schema: testing, Login: airflow, Password: XXXXXXXX, extra: None
[2023-10-01 09:51:52,447] {dbapi.py:180} INFO - Running statement: create table if not exists dags_run(
        dt  date,
        dag_id character varying,
        primary key (dt,dag_id)
        ), parameters: None
[2023-10-01 09:51:52,487] {dbapi.py:186} INFO - Rows affected: -1
[2023-10-01 09:51:52,508] {postgres.py:73} INFO - NOTICE:  relation "dags_run" already exists, skipping

[2023-10-01 09:51:52,626] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=postgres_v1, task_id=create_postgres_connection, execution_date=20210817T000000, start_date=20231001T095149, end_date=20231001T095152
[2023-10-01 09:51:54,760] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-01 09:51:55,168] {local_task_job.py:146} INFO - Task exited with return code 0
